js/newEddie.js

```diff
@@ -0,0 +1,15 @@
+var parents = document.getElementsByTagName("SELECT");
+var attr = ["selected", "disabled", "hidden"];
+var i;
+var j;
+
+for ( i=0; i<parents.length; i++){
+    
+    children = parents[i].firstElementChild;
```

Comment:
I couldn't agree more but I was trying to get rid of the "Move 'var' declaration to the top of the function" error. Which Vali told me later, it wasn't an error... more of a suggestion (i tried to conform because the code wasn't working, actually it was a spelling error). Solved